lib/index.js

```diff
@@ -1126,6 +1130,85 @@ class Mcdev {
         Util.logger.info(`:: Done\n`);
         return result;
     }
+    /**
+     * Updates notification email address field
+     *
+     * @param {string} businessUnit name of BU
+     * @param {TYPE.SupportedMetadataTypes} type limit execution to given metadata type
+     * @param {string[]} [keys] customerkey of the metadata
+     * @returns {Promise.<Object.<string, string[]>>} key: business unit name, value: list of affected item keys
+     */
+    static async updateNotifications(businessUnit, type, keys) {
+        const [cred, bu] = businessUnit ? businessUnit.split('/') : [null, null];
+        if (Util.OPTIONS.emailComplete == null && Util.OPTIONS.emailError == null) {
+            Util.logger.error(`No email addresses were provided`);
+            return null;
+        }
+        const properties = await config.getProperties();
+        const resultsObj = {};
+        if (bu === '*') {
+            Util.logger.info(':: Updating notifications on all BUs for all credentials');
+            let counter_credTotal = 0;
+            for (const cred in properties.credentials) {
+                Util.logger.info(`:: Updating notifications on BUs for ${cred}`);
+                let counter_credBu = 0;
+                for (const bu in properties.credentials[cred].businessUnits) {
+                    resultsObj[`${cred}/${bu}`] = await this.#updateNotificationsBU(
+                        cred,
+                        bu,
+                        type,
+                        keys
+                    );
+                    counter_credBu++;
+                    Util.startLogger(true);
+                }
+                counter_credTotal += counter_credBu;
+                Util.logger.info(`:: ${counter_credBu} BUs of ${cred}\n`);
+            }
+            Util.logger.info(`:: Updated notifications on ${counter_credTotal} BUs of ${cred}\n`);
+        } else {
+            resultsObj[`${cred}/${bu}`] = await this.#updateNotificationsBU(cred, bu, type, keys);
+        }
+
+        return resultsObj;
+    }
+    /**
+     *
+     * @param {string} cred name of Credential
+     * @param {string} bu name of BU
+     * @param {string} type metadata type
+     * @param {string[]} [keys] limit retrieval to given metadata keys
+     * @returns {Promise.<string[]>} keys of the automations where notifications were updated
+     */
+    static async #updateNotificationsBU(cred, bu, type, keys) {
```

Comment:
- [ ] lets please include this in `Mcdev.#runOnBU` like I did for `Mcdev.#fixKeys`.